src/default-retrieve-file-handler.ts

```diff
@@ -1,23 +0,0 @@
-import * as fs from 'fs';
-import { transpileIfTypescript } from './transpile-if-ts';
-
-export function defaultRetrieveFileHandler(path) {
-  // Trim the path to make sure there is no extra whitespace.
-  path = path.trim();
-
-  // This was removed because it seems that we can't use cache while expecting correct results
-  // TODO: check correctness and performance with file caching 
-  // if (path in fileContentsCache) {
-  //   return fileContentsCache[path];
-  // }
-
-  var contents: string;
-  try {
-    contents = fs.readFileSync(path, 'utf8');
-    contents = transpileIfTypescript(path, contents);
```

Comment:
Only this prevents me from pushing it to master.
@kulshekhar, could you, please, explain why we have to transpile code here one more time?